### DISCUSSION TITLE
What is indicators id?

### DISCUSSION THREAD

**Discussion by shankarcse2010** [Mathieu2301/TradingView-API#4] [Q&A] [answered]
Where can i find indicators id in tradingview ?
if i want use other indicators apart from sample code, where to find those ids and version?

**Comment by Mathieu2301** [comment, marked as answer]
You can use the "Network" tab of Chrome Inspector and look in the main websocket.

You can also use the ``market.searchIndicator()`` function.

**Reply by shankarcse2010** [reply]
Thanks @Mathieu2301
